token/core/common/certifier.go

```diff
@@ -13,21 +13,21 @@ import (
 
 // NewCertificationRequest creates a request to certify the tokens identified by the passed identifiers
 // zkatdlog does not make use of the certification service
-func (s *Service) NewCertificationRequest(ids []*token.ID) ([]byte, error) {
+func (s *Service[T]) NewCertificationRequest(ids []*token.ID) ([]byte, error) {
```

Comment:
Maybe there are reasons, but we could split the Service into smaller services, because these methods maybe don't necessarily share something in common with the rest of the struct.

Reply:
👍